extensions/extensions-helm/aissemble-policy-decision-point-chart/README.md

```diff
@@ -9,18 +9,19 @@ helm install policy-decision-point oci://ghcr.io/boozallen/aissemble-policy-deci
-| hostname                      | The hostname for the application            | No                | policy-decision-point                                                              |
-| image.name                    | The image name                              | Yes               | boozallen/aissemble-policy-decision-point                                          |
-| image.imagePullPolicy         | The image pull policy                       | No                | Always (ensures local docker image is pulled, rather than from Nexus repo)         |
-| image.dockerRepo              | The image docker repository                 | No                | ghcr.io/                                                                           |
-| image.tag                     | The image tag                               | No                | Chart.AppVersion                                                                   |
-| service.spec.ports            | The service spec ports                      | No                | - name: rest-api <br/>&emsp;&emsp;port: 8080 <br/>&emsp;&emsp;targetPort: 8080     |
-| deployment.ports              | The deployment ports                        | No                | - name: http-1 <br/>&emsp;&emsp;containerPort: 8080 <br/>&emsp;&emsp;protocol: TCP |
-| deployment.restartPolicy      | The deployment restart policy               | No                | Always                                                                             |
+| Property                     | Description                                                        | Required Override | Default                                                                            |
+|------------------------------|--------------------------------------------------------------------|-------------------|------------------------------------------------------------------------------------|
+| app.name                     | Sets label for app.kubernetes.io/name                              | No                | Chart.Name (aissemble-policy-decision-point-chart)                                 |
+| app.version                  | Sets label for app.kubernetes.io/version                           | No                | Chart.AppVersion (aiSSEMBLE project version)                                       |
+| hostname                     | The hostname for the application                                   | No                | policy-decision-point                                                              |
+| image.name                   | The image name                                                     | Yes               | boozallen/aissemble-policy-decision-point                                          |
+| image.imagePullPolicy        | The image pull policy                                              | No                | Always (ensures local docker image is pulled, rather than from Nexus repo)         |
```

Comment:
should this read "ensures local docker image is pulled, rather than from **ghcr.io** repo" instead of "Nexus repo"? Looks like we have a couple other `extensions-helm/*-chart/README.md` files that also still reference Nexus too. @ewilkins-csi

Reply:
Always means it will always pull from the remote repo, so this is not accurate as well